tests/acceptance/gcp-auth.bats

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bats
+
+#load _helpers
+#
+#SKIP_TEARDOWN=true
+VAULT_ADDR='http://127.0.0.1:8200'
```

Comment:
I believe this should be exported. I tested this with a "empty" set of env vars and I had to add `export` to this line for the vault client to be able to connect to the vault server.

Reply:
```suggestion
export VAULT_ADDR='http://127.0.0.1:8200'
```

Reply:
I am curious if you see the same? Maybe I have something misconfigured

Reply:
You're probably right, I realized I had already exported this variable through my zshrc file; which is likely why I wasn't seeing this error earlier

Reply:
Fixed in `ad20661`